sklearn/cross_validation.py

```diff
@@ -13,27 +12,29 @@
 from __future__ import division
 
 import warnings
-from itertools import chain, combinations
-from math import ceil, floor, factorial
 import numbers
-import time
+from itertools import combinations
 from abc import ABCMeta, abstractmethod
 
 import numpy as np
 import scipy.sparse as sp
+import sklearn.model_selection as mod_sel
```

Comment:
We also frown on "import as" for non standard imports: it makes reading the code harder because people need to scroll up to understand what the name of the module means.